docs/modules/module_check.rst

```diff
@@ -24,14 +24,13 @@ other ``text_formatter`` submodules.
 .. py:data:: text_formatter.check.strict_allowed_chars
 
    :type: str
-   :value: ``"ABCDEFGHIJKLMNOPKRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890"``
+   :value: ``"ABCDEFGHIJKLMNOPKRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890 "``
```

Comment:
I also had to document this.

Reply:
I consider that, when I am makinga user-facing change, the documentation must be updated on the same pull request, like this one.